src/coreclr/tools/StressLogAnalyzer/StressLogPlugin.cpp

```diff
@@ -1299,6 +1295,7 @@ int ProcessStressLog(void* baseAddress, int argc, char* argv[])
     memset(&mapImageToStringId, 0, sizeof(mapImageToStringId));
     memset(s_interestingStringFilter, 0, sizeof(s_interestingStringFilter));
     memset(s_interestingStringMatchMode, 0, sizeof(s_interestingStringMatchMode));
+    s_interestingStringMatchMode[IS_GCSTART] = true;
```

Comment:
This is an unfortunate detail to have here, but I don't think we want to generalize the `InterestingStrings` table to have a prefix bool.

Reply:
This will be a little easier to do in the managed implementation I think.